mapclassify/classifiers.py

```diff
@@ -2256,9 +2266,10 @@ class UserDefined(MapClassifier):
 
     """
 
-    def __init__(self, y, bins):
+    def __init__(self, y, bins, min=None):
```

Comment:
I'd rather use another keyword because now we're overriding built-in `min()`. Something like `legend_min` or `lowest`?